en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+= Invoke a Composer Flow 
+
+Invoke a Composer flow from external systems to enable deeper interoperability and integrated capabilities across Salesforce. You can also authenticate every invocable flow for a secure and protected automation experience, create API-compatible invocable actions for seamless integration with diverse systems and applications, and build powerful automations by leveraging the built-in support for input fields.
+
+When your invocable flow is activated, Composer exposes a unique flow URL and a unique pair of client credentials (client ID and client secret) so that your flow can be invoked from external systems. You can also reset these credentials for security best practices. Invocable flows are invoked by using an HTTP call to the flow URL and by using the flow client credentials as authentication. 
+
+To invoke a Composer flow, you must:
+
+. <<create-an-invocable-composer-flow, Create an invocable Composer flow.>> 
+. <<invoke-the-composer-flow-from-an-external-system, Invoke the Composer flow from an external system.>>
+
+[[create-an-invocable-composer-flow]]
+== Create an Invocable Composer Flow
+
+Create an invocable Composer flow so you can later invoke it from an external system. To create your invocable Composer flow, you must:
```

Comment:
```suggestion
Create an invocable Composer flow so that you can invoke it later from an external system:
```

Reply:
You don't need the second sentence since the first one can act as a lead-in. (Kim used to edit out the "To..." sentences in these cases.